addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -1708,13 +1708,18 @@ const RangeUserValueWidget = UnitUserValueWidget.extend({
         await this._super(...arguments);
         this.input = document.createElement('input');
         this.input.type = "range";
+        this.isCustomRange = !!this.el.dataset.values;
         let min = this.el.dataset.min && parseFloat(this.el.dataset.min) || 0;
         let max = this.el.dataset.max && parseFloat(this.el.dataset.max) || 100;
-        const step = this.el.dataset.step && parseFloat(this.el.dataset.step) || 1;
+        let step = this.el.dataset.step && parseFloat(this.el.dataset.step) || 1;
         if (min > max) {
             [min, max] = [max, min];
             this.input.classList.add('o_we_inverted_range');
         }
+        if (this.isCustomRange) {
+            this.customValues = JSON.parse(this.el.dataset.values);
```

Comment:
Let's make an easier syntax: instead of a JSON to parse, just make data-values a string that you have to split

```js
this.el.dataset.values.trim().split(/\s*,\s*/);
```